openslides_backend/action/actions/mediafile/permission_mixin.py

```diff
@@ -0,0 +1,139 @@
+from typing import Any, Dict, List, Optional, Tuple
+
+from ....permissions.management_levels import OrganizationManagementLevel
+from ....permissions.permission_helper import has_organization_management_level
+from ....services.datastore.commands import GetManyRequest
+from ....shared.exceptions import (
+    ActionException,
+    MissingPermission,
+    PermissionException,
+)
+from ....shared.filters import And, Filter, FilterOperator, Not
+from ....shared.patterns import KEYSEPARATOR, Collection, FullQualifiedId
+from ...action import Action
+
+
+class MediafilePermissionMixin(Action):
+    """
+    Mixin to handle the check_permissions of mediafile actions.
+    Overwrite check_permissions() and get_meeting_id().
+    """
+
+    def check_permissions(self, instance: Dict[str, Any]) -> None:
```

Comment:
This does way more than checking permissions. Everythign which doesn't throw a PermissionException should not be part of the `check_permissions` method, but of e.g. `update_instance`. Please move this accordingly and call `super().update_instance()` in the child classes, where necessary.

Reply:
I have moved the check code which throws ActionExceptions into a update_instance method of the mixin.